File: public/server/go/src/usr/testSleep.go
package usr

import (
	"GoPluginMagicModule/src/helper"

	"github.com/lockval/go2plugin"
)

func (e export) Export_testSleep(input *go2plugin.Input) map[string]any {

	//Execute "sys/testItsTime" after setting 3 seconds, pass parameter {n:2}
	input.Sleep(3000, "uuidXXXX", "sys/testItsTime", map[string]any{"n": 2})

	return helper.RespEmpty
}

File: public/server/javascript/src/usr/testSleep.ts
import { Dict, DBOperate } from "../libs/lockvalserver"

export function main(input: DBOperate<any>) {
	//Execute "sys/testItsTime" after setting 3 seconds, pass parameter {n:2}
	input.Sleep(3000, "uuidXXXX", "sys/testItsTime",{"n": 2})
  return {}
}
